spec/controllers/admin/roles_controller_spec.rb

```diff
@@ -3,7 +3,16 @@
 require 'rails_helper'
 
 RSpec.describe Api::V1::Admin::RolesController, type: :controller do
-  let(:user) { create(:user, role: create(:role, name: 'User')) }
+  let(:role) { create(:role, name: 'User') }
+  let(:user) { create(:user, role:) }
+  let(:manage_roles_permission) { create(:permission, name: 'ManageRoles') }
+  let!(:manage_roles_role_permission) do
+    create(:role_permission,
+           role_id: user.role_id,
+           permission_id: manage_roles_permission.id,
+           value: 'true',
+           provider: 'greenlight')
```

Comment:
```suggestion
    create(:role_permission,
           role: user.role,
           permission: manage_roles_permission,
           # if we rename manage_roles_permission -> permission that will further simplify it to `permission:`
           value: 'true',
           provider: 'greenlight')
```

Reply:
We have been using the wording "manage_roles_permission" instead of "permission" because in some cases, we need to add more than one permission to the test suite (ex "manage_users_permissions)

Reply:
Gotcha, thank you!